tests/conftest.py

```diff
@@ -16,6 +16,14 @@ class TestLevels(str, enum.Enum):
 
 DEFAULT_LEVEL = TestLevels.UNIT
 
+TEST_LEVEL_HIERARCHY = {
+    TestLevels.UNIT: 0,
```

Comment:
How about a singular form, TestLevel?

Reply:
wdym?

Reply:
My question is around

/s/TestLevels/TestLevel

?